### PR TITLE
README API documentation

### PR DESCRIPTION
**Pull request by Brunation11**
**--- Details ---**
- update API documentation with request perameters, response syntax, and guidance on sourcing request perameters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,27 +6,43 @@ The Amazon Connect Wisdom JavaScript library (WisdomJS) gives you the power to b
...
+
+The first step in setting up your Amazon Connect contact center is to create a virtual contact center instance. Each instance contains all the resources and settings related to your contact center. Follow the [Get started with Amazon Connect](https://docs.aws.amazon.com/connect/latest/adminguide/amazon-connect-get-started.html) user guide to get started.
+
+### Enable Amazon Connect Wisdom For Your Instance
+
+To utilize WisdomJS you should start by enabling Amazon Connect Wisdom for your Amazon Connect instance. Follow the [Enable Wisdom](https://docs.aws.amazon.com/connect/latest/adminguide/enable-wisdom.html) user guide to get started.
+
+### Set Up Integrated Applications
+
+All domains looking to intregrate with Amazon Connect and Amazon Connect Wisdom must be explicitly allowed for cross-domain access to the instance. For example, to integrate with your custom agent application, you must place your agent application domain in an allow list. To allow list a domain URL follow the [app integration guide](https://docs.aws.amazon.com/connect/latest/adminguide/app-integration.html).
```
Comment (briankropf-aws) [inline README.md:33]: typo 'intregrate' => 'integrate'

File: README.md [hunk 2]
```diff
@@ -42,7 +58,39 @@ const { Client, GetRecommendations } = require("amazon-connect-wisdomjs");
...
+  callSource: "agent-app",                                       // optional, defaults to 'agent-app'
+  serviceId: 'Wisdom',                                           // optional, defaults to 'Wisdom'
+  maxAttempts: 3,                                                // optional, defaults to 3
+  logger: {},                                                    // optional, if provided overrides default logger
+  headers: {},                                                   // optional, if provided overrides request headers
+  requestHandler: {},                                            // optional, if provided overrides the default request handler
+});
+```
+
+Initializing the WisdomJS client integrates with Connect by loading the pre-built Wisdom widget located at `<instanceUrl>/wisdom-v2` into an iframe and placing it into a container div. API requests are funneled through this widget and made available to your JS client code.
```
Comment (briankropf-aws) [inline README.md:80]: This reads strangely.  maybe drop 'Initializing' and just use 'The WIsdomJS client integrates...'

File: README.md (hunk 1 above)
Comment (Brunation11) [reply README.md:33]: nice catch ty.

File: README.md (hunk 2 above)
Comment (Brunation11) [reply README.md:80]: will do.

**Review by briankropf-aws — APPROVED** [review]
looks good to me

**Pull request merged**